plugins/modules/firewall.py

```diff
@@ -154,6 +154,16 @@
     allowlist_hos: yes
     rules:
       input:
+        - name: ICMP
```

Comment:
Yes, we might not need ICMP here.

Reply:
Thanks for your contribution! It would be helpful if you could expand the `name:` for the two new rules so they explain a bit more what they actually do.

Reply:
Updated.